onnxruntime/core/providers/acl/nn/pool.cc

```diff
@@ -63,11 +64,11 @@ ACLNEPool PoolOperation(onnxruntime::OpKernelContext* context,
     if (pool_attrs.global_pooling) {
       layer->configure(tpool.in.get(), tpool.out.get(), arm_compute::PoolingLayerInfo(pool_type));
     } else {
-      std::vector<int64_t> aclStrides(2);
+      TensorShapeVector aclStrides(2);
       aclStrides[0] = (strides.size() == 2) ? strides[1] : 1;
       aclStrides[1] = strides[0];
 
-      std::vector<int64_t> aclPads(4);
+      InlinedVector<int64_t, 4> aclPads(4);
```

Comment:
>, 4> [](http://example.com/codeflow?start=27&length=4)

Please, let InlinedVector to figure out the inlined size on its own. This would not compile if you replaced it back to std::vector.

Reply:
Done. In a previous version I tried without it and it didn't compile, so I thought it had to be explicit, but I see now that I had a different error causing the build fail.